src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -10,8 +10,7 @@
  */
 public class CommandResult {
 
-    /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```

Comment:
Nice to see that you encapsulated this field. However, do you think it is really necessary? 

Hint: Look at the original declaration.